hpcgap/lib/vecmat.gi

```diff
@@ -586,11 +586,6 @@ InstallOtherMethod( \[\], "for GF2 matrix",
       IsPosInt ],
     ELM_GF2MAT );
 
-InstallMethod( \[\], "for GF2 matrix",
-    [ IsGF2MatrixRep,
-      IsPosInt, IsPosInt ],
-    MAT_ELM_GF2MAT );
```

Comment:
This is now redundant as we already do `InstallMethod(MatElm, ..., MAT_ELM_GF2MAT);` later in the file.